packages/account-v2/src/components/SignupWizard/helpers.ts

```diff
@@ -0,0 +1,25 @@
+type TCustomStyles = {
+    content: ReactModal.Styles['content'];
+    overlay: ReactModal.Styles['overlay'];
+};
+
+export const CUSTOM_STYLES: TCustomStyles = {
```

Comment:
Is this a mock style file?

Reply:
These are the custom styles for `react-modal` if you've must have noticed being used to override the base styles for it.